README.md

```diff
@@ -170,9 +170,9 @@ Kubernetes users:\
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341) for known compatibility issues.
 
-## Compatibility with Jenkins >= 2.199
+## Compatibility with Jenkins >= 2.199 for JCasC < 1.36
```

Comment:
it's debatable if we even need this section at all anymore, I'm happy to release immediately after merging...

any thoughts?

Reply:
I thought about removing it when I was opening the PR, but finally I kept it because the new version needs to upgrade the baseline. If for any reason I cannot upgrade my jenkins, I'd like to find easily this information. I have no strong opinion since this would be in the source code of the past releases, but I, as a developer, would thank to keep it for 3-6 months (not forever of course). But just a matter of personal taste and any decision made here would fit me.